examples/Spirit/spirit_utils.cc

```diff
@@ -750,79 +750,87 @@ double calcTorqueInt(
 }
 
 template <typename T>
-void AddWorkCost(drake::multibody::MultibodyPlant<T> & plant,
+std::vector<drake::solvers::Binding<drake::solvers::Cost>> AddWorkCost(drake::multibody::MultibodyPlant<T> & plant,
```

Comment:
Can you explain what this change is doing?

Reply:
Is this related to the new non linear cost thing?

Reply:
The returned cost bindings allow us to use the solution and evaluate the cost coming from work.